workspace/__ardulib__/Graphics/Circle.inl

```diff
@@ -0,0 +1,56 @@
+
+CircleOutline::CircleOutline(XGraphics* parent) {
+    _parent = parent;
+}
+
+void CircleOutline::setPosition(int16_t x, int16_t y, int16_t r) {
+    _a = XVector2<int16_t>(x, y);
+    _r = r;
+}
+
+void CircleOutline::setStyle(XColor* strokeColor) {
+    _strokeColor = strokeColor;
+}
+
+void CircleOutline::renderScanline(XRenderer* renderer, int16_t scanline, uint16_t* buffer, size_t bufferSize) {
+
+    if (scanline < _a.y - _r || scanline > _a.y + _r)
+        return;
+
+    uint16_t color;
+
+    if (_strokeColor == nullptr)
+        color = xcolorTo565(getFGColor());
+    else
+        color = xcolorTo565(*_strokeColor);
+
+    putCircleOutlineOnScanline(_a, _r, scanline, buffer, bufferSize, color);
+}
+
+CircleSolid::CircleSolid(XGraphics* parent) {
+    _parent = parent;
+}
```

Comment:
Why override the base constructor if it does nothing new? I suspect you may completely drop it